two_factor/compat.py

```diff
@@ -11,27 +11,43 @@
     from django.contrib.formtools.wizard.views import SessionWizardView
     from django.contrib.formtools.wizard.storage.session import SessionStorage
 
+try:
+    from django.contrib.sites.shortcuts import get_current_site
+except ImportError:
+    from django.contrib.sites.models import get_current_site
 
-if django.VERSION[:2] >= (1, 8):
```

Comment:
By checking `django.VERSION` instead trying to import from a few locations it's easier to drop support for older Django versions. I'd rather keep checking `django.VERSION`, unless there's a good reason to change it.